components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/TokenMgtDAO.java

```diff
@@ -97,12 +98,12 @@
 
         try {
             String maxPoolSizeConfigValue = IdentityUtil.getProperty("JDBCPersistenceManager.SessionDataPersist" +
-                                                                     ".PoolSize");
+                    ".PoolSize");
             if (StringUtils.isNotBlank(maxPoolSizeConfigValue)) {
                 maxPoolSize = Integer.parseInt(maxPoolSizeConfigValue);
             }
         } catch (NumberFormatException e) {
-            if(log.isDebugEnabled()){
+            if (log.isDebugEnabled()) {
                 log.debug("Error while parsing the JDBCPersistenceManager.SessionDataPersist.PoolSize.", e);
```

Comment:
Better if you can log context information as well.

Reply:
done with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/472